base/int.jl

```diff
@@ -57,18 +57,28 @@ typemin(typeof(x))`, `abs(x) == x`, not `-x` as might be expected.
 abs(x::Signed) = flipsign(x,x)
 
 """
+    Base.checked_abs(x)
+
+The absolute value of `x`, with overflow error trapping where applicable.
+For types for which no overflow can happen during `abs`, this is equivalent
+to `abs(x)`.
+"""
+function checked_abs end
```

Comment:
Why have you defined this?

If you want to assign to the function you can do:

```
"docstring"
checked_abs
```
after the method is defined. But IIUC this is no longer any different to attaching it to a method.